incubator/vault/templates/deployment.yaml

```diff
@@ -63,9 +63,14 @@ spec:
         livenessProbe:
           # Alive if Vault is successfully responding to requests
           httpGet:
+          {{- if .Values.vault.enterprise }}
+            path: /v1/sys/health?performancestandbycode=204&
```

Comment:
```suggestion
            path: /v1/sys/health?perfstandbyok=true&
```

Reply:
I was running into issues when using the `perfstandbyok=true` status code. I'll do some testing and get back to you.